Makefile

```diff
@@ -12,7 +12,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/isolatedcontainers/sandboxed-containers-operator:4.8.0
+IMG ?= quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator:$(VERSION)
```

Comment:
Is it an OpenShift convention that the component name uses `_` and the namespace / container names contain `-`?

If I search for `openshift_` on quay.io, yours is the only org that shows up. If I search for `openshift-`, there are quite a few more, and at least that proves that `openshift-sandboxed-containers` would be a valid org name.

Reply:
It only worked with underscores when I created it, maybe a bug. It would be nice to change the org name but we can't do it ourselves we have to open a ticket with quay.io.